proposals/collection-literals.md

```diff
@@ -112,21 +112,18 @@ Collection literals are [target-typed](https://github.com/dotnet/csharplang/blob
 
     * The behavior of collection literals with collections that are not well-behaved is undefined.
 
-## Constructible collection types
+## Constructible collection types (target types)
 [constructible-collection-types]: #constructible-collection-types
 
 The following are _constructible collection types_ that can be used to construct collection literals.
-Actual translation of the literal to the corresponding is defined [below](#collection-literal-translation).
+These are the valid _target types_ for collection literals.
+Actual translation of the literal to the corresponding type is defined [below](#collection-literal-translation).
```

Comment:
can we add here (or elsewhere) that this list is ordered.  So if something supports multiple of these cases (for example, an array is both `T[]` *and* `IList<T>`, that the earlier one is preferred).  THis is also relevant for `types with a Constrct method` vs `collection initializers`.  `List<T>` may likely end up matching both, and we would want the `construct method` to win out over `collection initializer`.

Reply:
Added a note that the `Construct` method is used even if the type a _collection initializer_ type.

I don't think there's an ordering issue with `T[]` and `IList<T>` though because this list represents target types, where for `IList<T>`, the target type must be `IList<T>` rather than an implementation of `IList<T>`.

Reply:
good point.  thanks!